Automattic/newspack-plugin
# fix: temporarily disable query filtering from #2594

### All Submissions:






### Changes proposed in this Pull Request:



#2594 introduced query filters to allow `s` queries to apply to the custom credit field in Media Library searches. These filters break the query if another plugin is also adding a `JOIN` statement on `wp_postmeta`. This PR temporarily rolls back the changes in #2594 by simply removing the filter that modifies the default query, which should restore Media Library search functionality until we can properly fix the query modifications to play nice with other `JOIN` modifications.

### How to test the changes in this Pull Request:

1. On `release`, enable the Jetpack Videopress module in Jetpack > Settings > Modules.
2. Add an image block to a post or page and enter any search term in the search field. Observe that no results are returned, even if the search term should return matches, and that WP throws an error in the error log:

```
WordPress database error Not unique table/alias: 'wp_postmeta' for query 
```

3. Check out this branch and confirm that searching in the Media Library works again, albeit without the additional Credit field query introduced in #2594.

### Other information:
